dvc/fs/dvc.py

```diff
@@ -152,52 +149,11 @@ def repo_url(self):
         return self.repo.url
 
     @classmethod
-    def _repo_from_fs_config(
-        cls, **config
-    ) -> Tuple["Repo", Optional["RepoFactory"]]:
-        from dvc.external_repo import erepo_factory, external_repo
+    def _make_repo(cls, **kwargs) -> "Repo":
```

Comment:
Looks like we never create `DvcFileSystem` from `repo_url` before. So, looks like this is a vestige of ReferenceODB similar to #8276.